examples/land-cover-map/SI_LULC_pipeline.ipynb

```diff
@@ -2849,14 +2846,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# load sampled eopatches\n",
-    "eopatches = []\n",
-    "path_out_sampled = './eopatches_sampled'\n",
+    "# Load sampled eopatches\n",
+    "eopatch_samples = []\n",
     "\n",
-    "for idx in range(len(patchIDs)):\n",
-    "    eopatches.append(EOPatch.load(f'{path_out_sampled}/eopatch_{idx}', lazy_loading=True))    \n",
+    "for i in range(len(patchIDs)):\n",
+    "    sample_path = os.path.join(EOPATCH_SAMPLES_FOLDER, f'eopatch_{i}')\n",
+    "    eopatch_samples.append(EOPatch.load(sample_path, lazy_loading=True))    \n",
     "\n",
-    "eopatches = np.array(eopatches)"
+    "eopatch_samples = np.array(eopatch_samples)"
```

Comment:
I would completely remove this line. There is no reason for casting a list of EOPatches into a numpy array.